A/ADIOS2/build_tarballs.jl

```diff
@@ -75,10 +84,18 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    Dependency(PackageSpec(name="Blosc_jll")),
+    Dependency(PackageSpec(name="Bzip2_jll")),
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
     Dependency(PackageSpec(name="MPICH_jll")),
-    # Dependency(PackageSpec(name="MicrosoftMPI_jll")),
+    Dependency(PackageSpec(name="MicrosoftMPI_jll")),
+    Dependency(PackageSpec(name="Python_jll")),
```

Comment:
Looking at the [log](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=11038&view=logs&j=34cbdc54-f421-51b2-75f3-573b629b1abb&t=c354b55e-3752-50d7-7c13-81aa226ff98e&l=3206), it doesn't look like python is actually used? 
```
[22:37:23]       Python    : OFF
```

Reply:
Interesting. I'll remove Python. Maybe ADIOS2 needs some particular Python packages as well.